README.md

```diff
@@ -1,94 +1,187 @@
-# The ops library
+# The `ops` library
 
-<!-- The text below is also at the top of ops/__init__.py. Keep in sync! -->
 
-The ops library is a Python framework ([`available on PyPI`](https://pypi.org/project/ops/)) for developing
-and testing [Juju](https://juju.is/) charms in a consistent way, using standard Python constructs
-to allow for clean, maintainable, and reusable code.
+The `ops` library is a Python framework for developing and testing charms in a consistent way, using standard Python constructs to allow for clean, maintainable, and reusable code. It is an official component of the Charm SDK, itself a crucial part of the Juju universe.
 
-A charm is an operator -- business logic encapsulated in a reusable software
-package that automates every aspect of an application's life.
+> ✔️ `ops` is  ([`available on PyPI`](https://pypi.org/project/ops/)).
+>
+> ⚠️ The latest version of `ops` requires Python 3.8 or above.
+> 
+> 🥇 While charms can be written in any language, `ops` defines the latest standard. All new charms are encouraged to use it.
 
-Charms written with ops support Kubernetes using Juju's "sidecar charm"
-pattern, as well as charms that deploy to Linux-based machines and containers.
 
-Charms should do one thing and do it well. Each charm drives a single
-application and can be integrated with other charms to deliver a complex
-system. A charm handles creating the application in addition to scaling,
-configuration, optimisation, networking, service mesh, observability, and other
-day-2 operations specific to the application.
 
-The ops library is part of the Charm SDK (the other part being Charmcraft).
-Full developer documentation for the Charm SDK is available at
-https://juju.is/docs/sdk.
+||||
+|-|-|- |
+|| Learn about Juju! | Start deploying charms: <br> https://juju.is/docs/juju <br> _Juju is declarative, model-driven, and has native support for integrations and scaling, on any cloud. You want to deploy `foo`? Run `juju deploy foo`! You wish to integrate `foo` and `bar`? Run `juju integrate foo bar`!_ |
+||||
+|| Get better at Juju! | Take advantage of the charm ecosystem: <br> https://charmhub.io/ <br> _Quickly deploy a cluster ([OpenStack](https://charmhub.io/openstack-base), [Kubernetes](https://charmhub.io/charmed-kubernetes)), a database ([PostgreSQL](https://charmhub.io/postgresql-k8s), [MongoDB](https://charmhub.io/mongodb), etc.), an observability stack ([Canonical Observability Stack](https://charmhub.io/cos-lite)), an MLOps solution ([Kubeflow](https://charmhub.io/kubeflow)), and so much more!_ |
+||||
+|:point_right:| **Make Juju your own!** | Write your own charm: <br> https://juju.is/docs/sdk <br> _Juju is written in Go, but our SDK supports easy charm development in Python!_ |
 
-To learn more about Juju, visit https://juju.is/docs/olm.
+## Give it a try!
 
+Let's use `ops` to build a Kubernetes charm!
 
-## Pure Python
+### Set up
```

Comment:
I'm really not sure this section belongs here. I think we should link to something elsewhere, not repeat the relatively complicated setup on the `ops` README.

Reply:
I've restructured and updated the dev-setup doc so that we can straightforwardly link to it: https://juju.is/docs/sdk/dev-setup#heading--automatic-set-up-an-ubuntu-charm-dev-vm-with-multipass